docs/io/fits/api/verification.rst

```diff
@@ -2,24 +2,24 @@
 
 .. _verify:
 
-Verification options
+Verification Options
 ********************
 
-There are 5 options for the ``output_verify`` argument of the following methods
-of :class:`HDUList`: :meth:`~HDUList.close`, :meth:`~HDUList.writeto`, and
-:meth:`~HDUList.flush`, or the ``_BaseHDU.writeto`` method on any HDU
-object.  In these cases, the verification option is passed to a ``verify``
+There are five options for the ``output_verify`` argument of the following
+methods of :class:`HDUList`: :meth:`~HDUList.close`, :meth:`~HDUList.writeto`,
+and :meth:`~HDUList.flush`, or the ``_BaseHDU.writeto`` method on any HDU
+object. In these cases, the verification option is passed to a ``verify``
 call within these methods.
 
-exception
+Exception
```

Comment:
I think this and the headings below are actually indicating strings that can be passed, so I don't think the case should be changed. Instead, maybe these should be formatted like:
```suggestion
``'exception'``
```